src/sqlfluff/utils/reflow/respace.py

```diff
@@ -11,7 +11,8 @@
     RawSegment,
     WhitespaceSegment,
 )
-from sqlfluff.core.rules.base import LintFix, LintResult
+from sqlfluff.core.rules import LintFix
+from sqlfluff.core.rules.base import LintResult
```

Comment:
```suggestion
from sqlfluff.core.rules import LintFix, LintResult
```

Same comment as above...